spec/unit/enforcement_spec.rb

```diff
@@ -1,15 +1,29 @@
 # frozen_string_literal: true
+class MyController # < ApplicationController
```

Comment:
That's fine.  It's also clearly not important to any other piece of the stack.  In my opinion, it isn't important here.